### ISSUE TITLE
Semi-colons after class properties are not enforced

### ISSUE THREAD

**Issue by jethrolarson** [eslint/eslint#8641]
**Tell us about your environment**

* **ESLint Version:** 3.19.0
* **Node Version:** 6.2.1

**Parser** babel-eslint

**Configuration**
```
rules:
  semi: 1
  no-extra-semi: 1
```

```js
class Foo {
  bar = 1
  baz = 3;
}
```

**What did you expect to happen?**
Some kind of warning. Either for absence or presence of the semi-colon.

**What actually happened? Please include the actual, raw output from ESLint.**
No warnings shown.

I found [this plugin](https://github.com/marudor/eslint-plugin-class-property) that seems to address this but it seems like something that should be in core.

**Comment by not-an-aardvark** [comment]
Thanks for the issue. Class properties are experimental syntax, so they are not supported by ESLint (see our [policy on experimental features](https://github.com/eslint/eslint/blob/794d4d6c66f04c4afa6bdc2857a2f2880a57ccc9/README.md#what-about-experimental-features)). I would recommend a plugin instead.

**Comment by kaicataldo** [comment]
Specifically, the [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel) plugin has a forked version of the semi rule that should address this.

**Comment by not-an-aardvark** [comment]
Closing because this is not a bug with eslint.
